### PR TITLE
🧹  fix typo in docstring + remove LTTB from MinMaxLTTB + remove interleave_gaps

### PR DESCRIPTION
**Pull request by jonasvdd**

### CODE REVIEW

File: plotly_resampler/aggregation/aggregators.py [hunk 1]
```diff
@@ -243,11 +243,10 @@ def __init__(
         **downsample_kwargs
             Keyword arguments passed to the :class:`MinMaxLTTBDownsampler`.
             - The `parallel` argument is set to False by default.
-            - The `minmax_ratio` argument is set to 30 by default, whic was empirically
+            - The `minmax_ratio` argument is set to 4 by default, whic was empirically
```
Comment (jonasvdd) [inline plotly_resampler/aggregation/aggregators.py:246]:
```suggestion
            - The `minmax_ratio` argument is set to 4 by default, which was empirically
```

**Comment by jvdd** [comment]
still 1 job that failed over the nans...

File: plotly_resampler/aggregation/gap_handlers.py [hunk 2]
```diff
@@ -78,7 +78,8 @@ def _get_gap_mask(self, x_agg: np.ndarray) -> Optional[np.ndarray]:
         med_diff, x_diff = self._calc_med_diff(x_agg)
 
         # TODO: this 4 was revealed to me in a dream, but it seems to work well
-        gap_mask = x_diff > 4 * med_diff
+        # After some consideration, we altered this to a 4.1
+        gap_mask = x_diff > 4.1 * med_diff
         if not any(gap_mask):
```
Comment (jayceslesar) [inline plotly_resampler/aggregation/gap_handlers.py:83]: I arrived at almost the same number when I implemented this..funny how that works

**Comment by jonasvdd** [comment]
@jvdd, Do I handle here the Interleave gaps issue as well?

**Comment by jvdd** [comment]
> @jvdd, Do I handle here the Interleave gaps issue as well?

@jonasvdd sure!

**Comment by jonasvdd** [comment]
Okay @jvdd, decided to open a seperate PR which will be merged into this one w.r.t. the `interleave gaps`, see #204

**Comment by jonasvdd** [comment]
I think this PR can be merged as well @jvdd?

File: plotly_resampler/aggregation/gap_handlers.py (hunk 2 above)
Comment (jvdd) [reply plotly_resampler/aggregation/gap_handlers.py:83]: Still seems to fail here and there (but not consistently), but already significantly fewer test fails than with `4` :upside_down_face:

**Review by jvdd — APPROVED** [review]
LGTM!!

Still not entirely sure whether `4.1` is a perfect multiplier for our approximated `med_diff`.. Nonetheless, this (:cherries: :pick:) value seems to be an improvement over the `4` that was used before :magic_wand:  

Let's merge @jonasvdd :handshake:

**Pull request merged**